themes/grass/layouts/contribute/devel.html

```diff
@@ -40,11 +40,15 @@ <h3>Quick links</h3>
 
             </nav>
             <ul class="list-unstyled">
-	      <li><i class="fa fa-file-text"></i> &#160;<a href="https://grass.osgeo.org/programming7/" target="_blank">Developer manual</a></li>
-	      <li><i class="fa fa-github"></i> &#160;<a href="https://github.com/OSGeo/grass/" target="_blank"> Source code</a></li>
-	      <li><i class="fa fa-bug"></i> &#160;<a href="https://github.com/OSGeo/grass/issues" target="_blank"> Bug Tracker</a></li>
-              <li><i class="ms ms-grass-gis"></i> &#160;<a href="https://trac.osgeo.org/grass/wiki/HowToContribute" target="_blank"> How to contribute</a></li>
-            </ul>
+	            <li><i class="ms ms-grass-gis"></i> &#160;<a href="https://trac.osgeo.org/grass/wiki/HowToContribute" target="_blank">How to contribute</a></li>
+              <li><i class="fa fa-code"></i> &#160;<a href="https://github.com/OSGeo/grass/blob/master/CONTRIBUTING.md" target="_blank">How to start to code</a></li>
+              <li><i class="fab fa-git-alt"></i> &#160;<a href="https://trac.osgeo.org/grass/wiki/HowToGit" target="_blank">How to use git</a></li>
+      	      <li><i class="fa fa-file-text"></i> &#160;<a href="https://grass.osgeo.org/programming7/" target="_blank">Developer manual</a></li>
+	            <li><i class="fa fa-github"></i> &#160;<a href="https://github.com/OSGeo/grass/" target="_blank"> Source code</a></li>
+	            <li><i class="fa fa-bug"></i> &#160;<a href="https://github.com/OSGeo/grass/issues" target="_blank"> Bug Tracker</a></li>
+	            <li><i class="fas fa-passport"></i> &#160;<a href="https://grasswiki.osgeo.org/wiki/GRASS_GIS_APIs" target="_blank"> API's wiki page</a></li>
```

Comment:
I would move this up (second ?) as it is a very brief intro on the different ways you can code for GRASS.

Reply:
done in 454698c